src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/ClusterAwareConnectionProxy.java

```diff
@@ -1318,7 +1325,7 @@ private void logTopology() {
       msg.append("\n   [")
               .append(i)
               .append("]: ")
-              .append(hostInfo == null ? "<null>" : hostInfo.getHost());
+              .append(hostInfo.getHost());
```

Comment:
The previous code is safer.

Reply:
The only way the host would be null at this point is if we had a null entry in the topology list, which won't happen as far as I can tell since we removed the null writer implementation. However if you think I should add it back still I can do that